src/main/java/com/googleinterns/zoomtube/servlets/DiscussionServlet.java

```diff
@@ -86,19 +89,22 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     final Entity commentEntity;
     long lectureId = Long.parseLong(request.getParameter(PARAM_LECTURE));
     Key lecture = KeyFactory.createKey(LectureUtil.KIND, lectureId);
-    // TODO: Require non-zero content length. See: #183.
+    // TODO: Validate that the transcript line contains the timestampMs.
+    long transcriptId = Long.parseLong(request.getParameter(PARAM_TRANSCRIPT_LINE));
+    Key transcriptLine = KeyFactory.createKey(TranscriptLineUtil.KIND, transcriptId);
```

Comment:
Just a note, in some places we have 'key' in the name of the key. Like in `CommentUtil.java` this would be named `transcriptLineKey`. Can we stay consistent to avoid confusing in the naming, can be done in a seperate pr.

Reply:
Good point, I'll make the change in this one.  Thanks!